easypost/__init__.py

```diff
@@ -961,11 +961,10 @@ class Report(AllResource, CreateResource):
     def create(cls, api_key=None, **params):
         requestor = Requestor(api_key)
```

Comment:
we should add a docstring to this class, if not function

Reply:
I agree; however it would be the only docstring in the library. Could be a separate PR?